credits.txt

```diff
@@ -61,3 +84,187 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that make
+so much of this work possible.
+
+Proof Readers & Editors
```

Comment:
Editors & Proofreaders

Reply:
That does sound better. Made the change.